debian/rules

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	dh_auto_build
+	pod2man dh_package_notes $(CURDIR)/debian/dh_package_notes.1
+
+override_dh_auto_clean:
+	dh_auto_clean
+	rm -f $(CURDIR)/debian/dh_package_notes.1
```

Comment:
Have you considered using `debian/clean` instead?  Alternative, use the `execute_after_dh_auto_clean` target instead of the override.